### ISSUE TITLE
Krauss' sprites inconsistent

### ISSUE THREAD

**Issue by ChrisE87** [07th-mod/umineko-question#12]
It seems like Krauss' image switches back and forth between your version and the one from Steam depending on his pose, e.g. during his encounter with Kinzo.

http://imgur.com/a/OFYCz

**Comment by drojf** [comment]
I think @Forteissimo missed two of the sprites - it's just the wrong sprite in the folder `big\bmp\tati\kla\1`:

```
big\bmp\tati\kla\1\kla_nayamua1.png
big\bmp\tati\kla\1\kla_nayamua3.png
```

So this problem is easily fixed.

**Comment by ItaloKnox** [comment]
Fixed this on [v1.1](https://github.com/07th-mod/umineko-question/releases/tag/v1.1)
